### PR TITLE
Updated github actions.

### PR DESCRIPTION
**Pull request by lauri-codes**

### CODE REVIEW

**Comment by lauri-codes** [comment]
@ladinesa: Could you review and merge this? I just updated the repo to use latest github actions, project setup etc. to fix the CI problems.

**Review by ladinesa — DISMISSED** [review]

File: eelsdbparser/crawl.py
Comment (ladinesa) [inline eelsdbparser/crawl.py]: should we put a try here?

Comment (lauri-codes) [reply eelsdbparser/crawl.py]: Yes, I added it.

Comment (lauri-codes) [reply eelsdbparser/crawl.py]: Can you approve once you've taken a new look? Can't merge until approved.

**Review by ladinesa — APPROVED** [review]

**Pull request merged**
